docs/guide/install-and-run.asciidoc

```diff
@@ -1,113 +1,141 @@
 [[install-and-run]]
-== Install and run Elastic APM
+== Install and run
 
-To get started using Elastic APM,
-you need to have:
+This page is designed to make it easy to install and run the components of Elastic APM.
+Simply follow the step-by-step links below, and you'll be up and running in no time.
+Let's get started.
 
-* {apm-agents-ref}/index.html[APM agents] installed in your services
-* {apm-server-ref-v}/index.html[APM Server]
-* {ref}/index.html[Elasticsearch]
-* {kibana-ref}[Kibana]
+* <<before-installation>>
+* <<install-elasticsearch>>
+* <<install-kibana>>
+* <<apm-server>>
+* <<kibana-dashboards>>
+* <<agents>>
 
-IMPORTANT: For best results, ensure you're using the same version of Elasticsearch, Kibana, and APM Server.
-
-First, you'll need to install and start an Elasticsearch cluster and Kibana. Information on how to do this can be found in the {stack-gs}/get-started-elastic-stack.html[Elastic Stack getting started guide]. Specifically, the {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[install Elasticsearch], and {stack-gs}/get-started-elastic-stack.html#install-kibana[install Kibana] sections.
-
-The following sections show how to get started quickly with Elastic APM on a local machine.
-
-[[apm-server]]
 [float]
-=== Install and run APM Server
+[[before-installation]]
+=== Before you begin
 
-First, https://www.elastic.co/downloads/apm/apm-server[download APM Server] for your operating system and extract the package.
+* See the https://www.elastic.co/support/matrix[Elastic Support
+Matrix] for information about supported operating systems and product
+compatibility.
+* Verify that your system meets the
+https://www.elastic.co/support/matrix#matrix_jvm[minimum JVM requirements] for {es}.
 
-In a production environment you would put APM Server on its own machines,
-similar to how you run Elasticsearch.
-You _can_ run it on the same machines as Elasticsearch,
-but this is not recommended,
-as the processes will be competing for resources.
+IMPORTANT: For best results, ensure you’re using the same version of Elasticsearch, Kibana, and APM Server.
```

Comment:
This `For best results` implies that version consistency is optional but nice, when in most cases is required. I'll remove that